roles/pf9-hostagent/files/pf9_get_hostagent.py.j2

```diff
@@ -16,12 +16,18 @@ elif sys.version_info.major == 3:
 def do_request(action, host, relative_url, headers, body, proxy=""):
     if proxy != "":
        proxyData = proxy.split(":")
-       proxyHost = proxyData[0]
-       proxyPort = proxyData[1]
+       if len(proxyData) < 3:
+           proxyHost = proxyData[0]
+           proxyPort = proxyData[1]
+       elif len(proxyData) > 2:
+           proxyHost = proxyData[1]
+           proxyPort = proxyData[2]
+       else:
+           print "Error inccorect proxy configuration"
```

Comment:
Couple of things here...

i assume this is to split a URL with http/s://<proxy>:<port> OR just <proxy>:<port>? Won't proxyData[1] end up being '//<proxyaddr>'?

Also misspelling on line 26.